Source/inv.cpp

```diff
@@ -888,7 +888,12 @@ void CheckInvCut(int pnum, Point cursorPosition, bool automaticMove, bool dropIt
 	}
 
 	if (dropItem) {
-		TryDropItem();
+		if (TryDropItem()) {
+			// Reset cursor position
+			SetCursorPos(cursorPosition);
+		} else {
+			player.Say(HeroSpeech::WhereWouldIPutThis);

```

Comment:
Shouldn't this go inside the `TryDropItem` method instead?

Where else is that method being used today? Do we emit the same sound in there?

Reply:
Should we consider randomizing this with something like `ICantDoThat`?

Reply:
> Shouldn't this go inside the `TryDropItem` method instead?
> 
> Where else is that method being used today? Do we emit the same sound in there?

@julealgon this method is called in a few places, but currently, only misc_msg.cpp `BlurInventory()` plays the audio if the result is false.

Reply:
It seems wrong to me that we are not (apparently) consistently playing the same audio in the same situations. Maybe @AJenbo can comment here since (I think?) `TryDrop` was created for gamepad controls initially.

Reply:
Yes, make it always say the line when it fails.